### PR TITLE
Feature/app 5530 update transaction list item

### PR DESCRIPTION
**Pull request by acassiovilasboas**
## Description
- add chevron TransciontListItem component
- fix Typografy skeleton

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

### Dinamic skeleton typografy
Simulator Screenshot - iPhone 15 - 2024-03-15 at 16 36 11

### TransctionListItem with chevron
simulator_screenshot_8F7D6B60-1E02-4236-825B-645CD6AB2DC7

## Checklist:

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) document
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have made corresponding changes to the documentation (if appropriate)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] All new and existing tests passed

### CODE REVIEW

File: OceanComponents/Classes/Extensions SwiftUI/EdgeInsets+Extensions.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//  OceanComponents
+//
+//  Created by Acassio Mendonça on 15/03/24.
+//
+
+import Foundation
+import SwiftUI
+
+extension EdgeInsets {
+    public static func all(_ size: CGFloat) -> Self {
```
Comment (vromeiro) [inline OceanComponents/Classes/Extensions SwiftUI/EdgeInsets+Extensions.swift:12]: 💙

**Review by vromeiro — APPROVED** [review]

File: OceanComponents/Classes/Components SwiftUI/Typography/OceanSwiftUI+Typography.swift [hunk 2]
```diff
@@ -64,6 +64,8 @@ extension OceanSwiftUI {
 
         // MARK: Properties private
 
+        @State private var labelWidth: CGSize = .zero
```
Comment (renanmassaroto) [inline OceanComponents/Classes/Components SwiftUI/Typography/OceanSwiftUI+Typography.swift:67]: Acho que não tá mais usando, né? Se não tiver, pode apagar.

File: Podfile.lock [hunk 3]
```diff
@@ -54,4 +54,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ab8e24fd7284a70c6fd7c94aac7243e86a16da07
 
-COCOAPODS: 1.15.2
+COCOAPODS: 1.13.0
```
Comment (renanmassaroto) [inline Podfile.lock:57]: Eeeeee

**Review by renanmassaroto — APPROVED** [review]

File: OceanComponents/Classes/Components SwiftUI/Typography/OceanSwiftUI+Typography.swift (hunk 2 above)
Comment (acassiovilasboas) [reply OceanComponents/Classes/Components SwiftUI/Typography/OceanSwiftUI+Typography.swift:67]: hahahahahaha, boa

File: Podfile.lock (hunk 3 above)
Comment (acassiovilasboas) [reply Podfile.lock:57]: haaaaaaa

**Pull request merged**